### PR TITLE
Add remediation content

### PR DESCRIPTION
**Pull request by nelsonkopliku**
# Description

Adds the remediation content in the dialog that opens when clicking on a check row in ExecutionResults

remediation-suggestion-dialog

## How was this tested?

Automatic test added

### CODE REVIEW

**Review by arbulu89 — APPROVED** [review]
Nice catch on this @nelsonkopliku

File: assets/js/components/ChecksResults/checksUtils.js [hunk 1]
```diff
@@ -109,3 +109,8 @@ export const getCheckDescription = (catalog, checkID) => {
   }
   return null;
 };
+
+export const getCheckRemediation = (catalog, checkID) => {
+  const check = findCheck(catalog, checkID);
+  return check?.remediation;
```
[the remark this answers is not in the document]
Comment (nelsonkopliku) [reply assets/js/components/ChecksResults/checksUtils.js:115]: i guess so :smile: But if any preference I can change it to `if whathever`

**Comment by arbulu89** [comment]
By the way @nelsonkopliku , I noticed in the gif that we are not rendering the modal title as markdown. Yet another good to have fix hehe

File: assets/js/components/ChecksResults/checksUtils.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (nelsonkopliku) [reply assets/js/components/ChecksResults/checksUtils.js]: I changed to an `if` for consistency with `getCheckDescription`

**Comment by nelsonkopliku** [comment]
@arbulu89 made also the dialog title a markdown. Thanks! I missed it :rocket:

**Review by EMaksy — COMMENTED** [review]
LGTM!

Just a small idea about the Modal component.
For now we need to click somewhere outside the Modal, this may confuse the user on the first usage.
What do you think to add an Exit/Close button to it ?

**Comment by nelsonkopliku** [comment]
> What do you think to add an Exit/Close button to it ?

@EMaksy I think it is a good idea, however, since we are going to revisit this soon I'd keep the close button  as part of that improvement. Makes sense?

**Pull request merged**